akka-samples/akka-sample-persistence-scala/build.sbt

```diff
@@ -6,5 +6,7 @@ scalaVersion := "2.11.7"
 
 libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % "2.4-SNAPSHOT",
-  "com.typesafe.akka" %% "akka-persistence" % "2.4-SNAPSHOT"
```

Comment:
I remember that sample build transformer does not support transitive dependencies when transforming from `libraryDependencies` to `projectDependencies`. That is why we need to explicitly declare all akka dependencies in samples. In that case `akka-sample-persistence-java` should not compile. But it does. :)